esp-hal-common/Cargo.toml

```diff
@@ -16,6 +16,7 @@ bitflags             = "2.3.3"
 bitfield             = "0.14.0"
 cfg-if               = "1.0.0"
 critical-section     = "1.1.2"
+defmt                = { version = "=0.3.5", optional = true }
```

Comment:
Is there a reason to fix the version to 0.3.5?

Reply:
yes, I'd love to minimize headache caused by their msrv policy

Reply:
Ah, fair enough!